recipes-connectivity/access-point/access-point.bb

```diff
@@ -0,0 +1,36 @@
+SUMMARY = "Deployment of Access Point configurations"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+RDEPENDS:${PN} = "hostapd"
+
+S = "${WORKDIR}"
+
+SRC_URI = " \
+    file://access-point.service \
+    file://access-point.network \
+    file://access-point.conf \
+"
+
+inherit allarch systemd
+
+SYSTEMD_SERVICE:${PN} = "access-point.service"
+SYSTEMD_AUTO_ENABLE:${PN} = "disable"
```

Comment:
Does this mean it is disabled by default? Then it's ok. Access point mode should not be enabled by default

Reply:
Yes, it is disabled by default.